spec/unit/govuk_index/presenters/indexable_content_presenter_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'spec_helper'
+
+RSpec.describe GovukIndex::IndexableContentPresenter do
+  it "details_with_govspeak_and_text_html" do
+    details = {
+      "body" => [
+        { "content_type" => "text/govspeak", "content" => "**hello**" },
+        { "content_type" => "text/html", "content" => "<strong>hello</strong>" }
+      ]
+    }
+
+    expect(indexable_content_presenter(details).indexable_content).to eq("hello")
+  end
+
+  it "indexes contact details correctly" do
+    details = {
+      "some_other_key" => "value",
+      "title" => "Title",
+      "description" => "Description",
+      "contact_groups" => [
+        {
+          "slug" => "slug-1",
+          "title" => "Title 1"
+        },
+        {
+          "slug" => "slug-2",
+          "title" => "Title 2"
+        }
+      ]
+    }
+
+    expect(indexable_content_presenter(details, 'contact').indexable_content).to eq("Title\n\n\nDescription\n\n\nTitle 1\n\n\nTitle 2")
```

Comment:
How come there's so many newlines between stuff? I don't think it matters for searching as it will get split into tokens regardless, but seems a bit strange.